routes.js

```diff
@@ -0,0 +1,14 @@
+const routes = module.exports = require('next-routes')()
+
+routes
+  // Account
+  .add('account confirmation', '/confirmation/:token', '/confirmation')
+
+  // Characters
+  .add('character profile', '/my/characters/:id', '/my/character')
+
+  // Groups
+  .add('create group', '/groups/create', '/groups/create')
+  .add('manage group', '/groups/manage', '/groups/manage')
+  .add('group search', '/groups/search', '/groups/search')
+  .add('group profile', '/groups/:id', '/groups/group')
```

Comment:
I'm pretty sure that there's a way for us to pass in a param that will be parsed as regex, then we can just validate that the parameter is a UUID. I wasn't able to figure that out this morning, though.

Reply:
I believe that `next-routes` uses [this package](https://github.com/pillarjs/path-to-regexp]) to parse paths. So anything you put in as a path can follow that project's rules.